edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedKubernetesConfigProvider.cs

```diff
@@ -108,5 +108,132 @@ static string BindPath(Uri uri)
                 ? Path.GetDirectoryName(uri.LocalPath)
                 : uri.AbsolutePath;
         }
+
+        void InjectNetworkAliases(IModule module, CreateContainerParameters createOptions)
+        {
+            if (createOptions.NetworkingConfig?.EndpointsConfig == null)
+            {
+                string networkId = this.configSource.Configuration.GetValue<string>(Core.Constants.NetworkIdKey);
+                string edgeDeviceHostName = this.configSource.Configuration.GetValue<string>(Core.Constants.EdgeDeviceHostNameKey);
+
+                if (!string.IsNullOrWhiteSpace(networkId))
+                {
+                    var endpointSettings = new EndpointSettings();
+                    if (module.Name.Equals(Core.Constants.EdgeHubModuleName, StringComparison.OrdinalIgnoreCase)
+                        && !string.IsNullOrWhiteSpace(edgeDeviceHostName))
+                    {
+                        endpointSettings.Aliases = new List<string>
+                        {
+                            edgeDeviceHostName
+                        };
+                    }
+
+                    IDictionary<string, EndpointSettings> endpointsConfig = new Dictionary<string, EndpointSettings>
+                    {
+                        [networkId] = endpointSettings
+                    };
+                    createOptions.NetworkingConfig = new NetworkingConfig
+                    {
+                        EndpointsConfig = endpointsConfig
+                    };
+                }
+            }
+        }
     }
+
+    // public class CombinedKubernetesConfigProvider1 : CombinedDockerConfigProvider
```

Comment:
Cleanup?

Reply:
This has been removed by later changesets